docs/book/response.md

```diff
@@ -138,3 +138,16 @@ $response->setContent(<<<EOS
 </html>
 EOS);
 ```
+
+### handle Headers already sent
+
+Since 2.13.0, we can handle header already sent by pass callable via `Response::setHeadersSentHandler()`:
```

Comment:
## Version hints

If a new feature was added with a minor release then a version hint must be added to the related section of the documentation page.
The hint must be added after the headline of the new feature.

## Syntax example

``````
> Available since version 2.13.0
``````

Reply:
What section that related? "Available Methods" part?

Reply:
@samsonasik 
> The hint must be added after the headline of the new feature.

Reply:
updated.